stable/prometheus-operator/values.yaml

```diff
@@ -667,13 +667,20 @@ prometheus:
     ## to load rules
     ##
     ruleSelector: {}
```

Comment:
Hey @paskal, since you're cleaning up the comments, I think it's worth cleaning up the comment above also. It's copy-pasted from the coreos implementation and the rules in this chart don't have `rules.additionalLabels`.

Reply:
Is current state is OK?